### PR TITLE
Request vp media type

### PR DESCRIPTION
**Pull request by OR13**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1144.html" title="Last updated on Jun 7, 2023, 5:57 PM UTC (96af42f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1144/e6e9fbf...96af42f.html" title="Last updated on Jun 7, 2023, 5:57 PM UTC (96af42f)">Diff</a>

### CODE REVIEW

**Review by andresuribe87 — APPROVED** [review]

**Review by awoie — APPROVED** [review]
If VPs stay in the spec, then this lgtm.

**Review by brentzundel — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-06-07-vcwg#section1-2) on 2023-06-07

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.2. JWT PRs.

**Orie Steele:** There are 2 new PRs on VC-JWT, 103 and 104.  

_See github pull request [vc-jwt#103](https://github.com/w3c/vc-jwt/pull/103)._




_See github pull request [vc-jwt#104](https://github.com/w3c/vc-jwt/pull/104)._




**Orie Steele:** There are 2 new PRs on VC-JWT, 103 and 104.  

_See github pull request [vc-data-model#1144](https://github.com/w3c/vc-data-model/pull/1144)._




**Orie Steele:** PRs up in the core data model 1144.  

> *Phillip Long:* pdl-ASU has joined #vcwg.

**Joe Andrieu:** Added a comment to 103. Would it be useful to reuse the pattern for VCs. A base media type plus securing?  
… both securing mechanisms will need a payload with a common basis.  

> *Dmitri Zagidulin:* +1 Joe, I was wondering about that as well.

> *Orie Steele:* correct, that is what it is doing.

**Dave Longley:** agrees it is following the same pattern.  

**Orie Steele:** talking about confidence method in registry.  

> *Dave Longley:* +1 to adding confidence method to the reserved terms table + v2 context, sounds good.

**Brent Zundel:** 9 open PRs, please review and look thru them.  
… moving on to final topic, issues.  

**Ted Thibodeau Jr.:** suggest that we call out specific issues and PRs in agenda ahead of time.  

> *Orie Steele:* There are also several open PRs here: [https://github.com/w3c/vc-status-list-2021/pulls](https://github.com/w3c/vc-status-list-2021/pulls) which can be discussed.

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-06-14-vcwg#section1-3) on 2023-06-14

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.3. Request vp media type (pr vc-data-model#1144)

_See github pull request [vc-data-model#1144](https://github.com/w3c/vc-data-model/pull/1144)._




**Manu Sporny:** pull 1144 adds a media type for vps.  
… expect that to merge soon.  

</details>

File: index.html [hunk 1]
```diff
@@ -5167,6 +5167,61 @@ <h2>The <code>application/vc+ld+json</code> Media Type</h2>
...
+
+  <p>
+    Note that while the Verifiable Credentials format uses JSON-LD
+    conventions, there are a number of constraints and additional
+    requirements for Verifiable Credential implementations that justify
+    the use of a specific media type.
+  </p>
+
+  <p>
+    This media type can be used with credentials secured using an <a>external proof</a>.
```
Comment (msporny) [inline index.html:5220]:
```suggestion
    This media type can be used with credentials secured using an <a>external proof</a>.
```

Hmm, this looks wrong? Is this what we're trying to say? I'm going to merge this, but wondering if this is a copy-paste error. If so, @OR13, can you fix it in a future (editorial) PR?

**Comment by msporny** [comment]
Normative, multiple reviews, no changes requested, no objections, merging.

**Pull request merged**